source/includes/_queries.md

```diff
@@ -40,13 +41,22 @@ Each query consists of following things:
     <li> **Query object** might be any of types, described below </li>
 </ul>
 
-**Signatures** contain one or many signatures (ed25519 pubkey + signature):
+**Signatures** contain one or many signatures (ed25519 pubkey + signature).
+
+## Validation
+
+The validation for all commands includes:
+
+ * timestamp — it should be in more than time of the peer minus 24 hours
+ * signature of query creator — is used for checking the identity of query creator
```

Comment:
"is" is unnecessary as we already have -

Reply:
ok